packaging/wheel/osx_wheel.sh

```diff
@@ -39,7 +39,7 @@ do
     echo "Building against ${TORCHAUDIO_PYTORCH_DEPENDENCY_VERSION}"
 
     # install torchvision dependencies
-    pip install ninja scipy pytest
+    pip install ninja scipy pytest rarfile
```

Comment:
We should not make `rarfile` a required dependency of torchvision when building the wheels, as only installing `rarfile` is not enough (you also need the required dependencies to be installed).